nginx/sites-enabled/aapinstaller.http_and_https.conf.disabled

```diff
@@ -42,16 +42,26 @@ server {
 
     # main location for any remaining URI, checks for session cookie
     location / {
-
         # if madd_session cookie is not set, redirect to login
+        # TODO SSO - Temporarily overriding to send directly to SSO login
```

Comment:
Can remove nginx changes before merging and leave for the UI update PR

Reply:
This is ok, it will be taken care of in different PR.